### PR TITLE
Improvement/bandswidget sizewarning

### PR DESCRIPTION
**Pull request by AndresOrtegaGuerrero**
This PR address #820 

The idea I have is to include a javascript that just check if the width and display a message if the size is too narrow. 

image

I am not well knowledgable in JavaScript ( I got a lot of help on this one from ChatGPT) Please let me know any suggestions or ideas you might have in mind to address this issue ? 

@edan-bainglass @superstar54

### CODE REVIEW

**Comment by edan-bainglass** [comment]
Using CSS, you could modify the overflow-x property of the plot's container. This will add a horizontal scroll bar when the full container does not fit the screen.

**Comment by AndresOrtegaGuerrero** [comment]
@edan-bainglass Something like this ?

image

Now the question is showing the scrolling horizontal bar is enough , or if there is the need of the warning displayed as well.

@cpignedoli

**Comment by edan-bainglass** [comment]
Looks good. Enough in my option.

**Comment by AndresOrtegaGuerrero** [comment]
> Using CSS, you could modify the overflow-x property of the plot's container. This will add a horizontal scroll bar when the full container does not fit the screen.

The only thing is that the current PR , doesnt contain this behaviour of hide or display the horizontal bar ...

**Comment by AndresOrtegaGuerrero** [comment]
Now the question , is should we do it only for electronic structure , or include PDOS tab as well ?

**Comment by superstar54** [comment]
> Now the question , is should we do it only for electronic structure , or include PDOS tab as well ?

If possible, it would be good for all the widgets to have a scroll bar on a small screen.

File: src/aiidalab_qe/plugins/electronic_structure/result.py [hunk 1]
```diff
@@ -37,5 +39,20 @@ def _update_view(self):
...
+        )
+        _bands_dos_widget.layout = ipw.Layout(width="1000px")
+        self.children = [
+            ipw.VBox(
+                [
+                    plot_container,  # The scrollable container
+                    _bands_dos_widget,  # The actual Plotly widget
+                ]
+            )
+        ]
```
[the remark this answers is not in the document]
Comment (AndresOrtegaGuerrero) [reply src/aiidalab_qe/plugins/electronic_structure/result.py:58]: But then the bar is enough , or do we want a warning too ?

[the remark this answers is not in the document]
Comment (superstar54) [reply src/aiidalab_qe/plugins/electronic_structure/result.py:58]: Ah, the `plot_container` is used for the warning message. Sorry, I misunderstood it.

I think the bar is enough.

**Review by superstar54 — APPROVED** [review]
Thanks for the work! LGTM!

**Pull request merged**